rasa/shared/core/events.py

```diff
@@ -314,21 +317,30 @@ def apply_to(self, tracker: "DialogueStateTracker") -> None:
 
     def __eq__(self, other: Any) -> bool:
```

Comment:
We can also make this an `abstractmethod` now right?

Reply:
actually didn't know about this decorator. Added it 👍🏻